js/config-schema.json

```diff
@@ -98,14 +98,28 @@
         "CALC_TREND": {
           "type": "boolean",
           "default": false
+        },
+        "ALWAYS_ON_TOP": {
+          "type": "boolean",
+          "default": true
         }
       },
-      "required": [
-        "AGE_LIMIT",
-        "POSITION",
-        "UNITS_IN_MMOL",
-        "CALC_TREND"
-      ]
+      "required": ["AGE_LIMIT", "POSITION", "UNITS_IN_MMOL", "CALC_TREND"]
```

Comment:
This is not backward-compatible. Once you run your version, it will add new properties to the config that aren't allowed by the scheme of the previous version. It will cause the config error on every run.
<img width="300" alt="Screenshot 2025-01-27 at 23 22 51" src="https://github.com/user-attachments/assets/5ecf4f10-645a-4c20-8452-3daa37b6f65e" />